crates/view/src/service.rs

```diff
@@ -747,7 +747,7 @@ impl ViewService for ViewServer {
                     tonic::Status::invalid_argument(format!("Could not parse pair: {e:#}"))
                 })?;
 
-            planner.position_withdraw(position_id, reserves, trading_pair);
+            planner.position_withdraw(position_id, reserves, trading_pair, 0);
```

Comment:
Why is this being 0 ok? Wouldn't we want to follow the planning request?

Reply:
This is kind of a fake TPR implementation, unlike the one in Prax. It's not used by the regular pcli commands, I think an integration test uses it maybe. Making it work here is possible but low ROI, so I opted for not doing anything

Reply:
Sounds good ; we could maybe add a comment to that effect